packages/common/core-interfaces/src/logger.ts

```diff
@@ -30,8 +51,15 @@ export interface ITaggedTelemetryPropertyType {
 /**
  * JSON-serializable properties, which will be logged with telemetry.
  */
+export type ITelemetryBaseProperties = ITelemetryProperties;
+
+/**
+ * @see {@link ITelemetryBaseProperties}
```

Comment:
Nit: I think we can still use `inheritDoc` here and add the `deprecated` tag.
```suggestion
 * {@inheritDoc ITelemetryBaseProperties}
```

Reply:
Cool I'll try it.  Thanks for all the tips!

Reply:
Ok yeah that got rid of the "undocumented" flag in the api.md.  I wish VSCode respected `@inheritDoc`, is there a plugin that adds that or something?

Reply:
It's really inconsistent. It often works, but doesn't always. I _think_ VSCode assumes that there should be no other contents in the comment other than the `@inheritDoc` comment, and when there is it doesn't do the substitution.

Reply:
I haven't looked into plugins, but that's an interesting thought.